source/bson-corpus/tests/top.json

```diff
@@ -69,11 +69,11 @@
     "parseErrors": [
         {
             "description" : "Bad $regularExpression (extra field)",
-            "string" : "{\"a\" : \"$regularExpression\": {\"pattern\": \"abc\", \"options\": \"\", \"unrelated\": true}}}"
+            "string" : "{\"a\" : {\"$regularExpression\": {\"pattern\": \"abc\", \"options\": \"\", \"unrelated\": true}}}"
```

Comment:
_wow_ how did we miss this! I guess test runners are just verifying that _some_ error is thrown?

Reply:
That's my guess.  I happened to have a test looking for a more specific error and found this bug.